contrib/format-ltsv/src/main/resources/bootstrap-format-plugins.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Nice change. These new files add bootstrap config for several config plugins. Are there entries for these plugins in the "master" bootstrap file that should be removed?

Reply:
No, they were not present.